tests/checks/common.py

```diff
@@ -33,6 +33,10 @@
 AUTO_CONFD_PATH = os.path.join(os.path.dirname(os.path.realpath(__file__)), '..', '..', 'conf.d', 'auto_conf')
 
 def copy_checks():
+    if not os.path.exists(CHECKSD_PATH):
```

Comment:
I think it's reasonable to do something else now that the folders no longer exist, though when it did it made sense to keep doing this.

We should at least look into it, it's not a bad idea. Let's make a card.